### PR TITLE
Implement create a calendar event func

### PR DESCRIPTION
**Pull request by wiseaidev**
## PR Acceptance Criteria
Related to #44 

## Guidelines
* [X] PR follows the [Contributing guidelines](https://github.com/nylas-samples/nylas-hacktoberfest-2023/blob/main/CONTRIBUTION.md)?
* [X] PR Adheres to our [Code of Conduct](https://github.com/nylas-samples/nylas-hacktoberfest-2023/blob/main/CODE_OF_CONDUCT.md)?
* [X] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Code complete
* [X] A `README.md` available describing how to setup and test the code?
* [X] The code sample in the correct location (i.e. `calendar-event/python/create_calendar_event`)?
* [x] Link added to the repository readme (view `# Code Snippets created by the Nylas Community` with examples) describing what the code sample does for others to try out
* [x] The code can be run using Github Codespaces, so the maintainer can try out the code sample remotely to ensure everything is working before we accept the PR.

Screenshot from 2023-10-23 11-53-55

## For PR Reviewer use
* [x] Purpose of code sample is clear
* [x] The code sample is readable
* [x] Steps try out the code are provided
* [x] Code sample runs on codespaces environment
* [x] Code works as expected
* [x] PR labels added

### CODE REVIEW

**Comment by atejada** [comment]
Hello @wiseaidev I send you an email with the details on how to finish setting up your V3 account. Once you do that, please update your source code so that it can be tested and accepted.

File: code-samples/calendar/python/create_calendar_event/create_calendar_event/__main__.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        >>> event = create_calendar_event(client, 'Meeting', 'Conference Room', 1729457999, 1729458000, participants, 'your_calendar_id' )
+
+        Create an event with a description and participants and save it in a specific calendar:
+        Create an event with a description and participants and save it in a specific calendar:
+        >>> participants = [{'email': 'oss1@wiseai.dev'}, {'email': 'oss2@wiseai.dev'}]
+        >>> event = create_calendar_event(client, 'Team Meeting', 'Virtual', 1729457999, 1729458000, 'Discuss project progress', participants, 'your_calendar_id')
+    """
+    try:
+        # create an event for that calendar ID
+        # request_body = CreateEventRequest(title=title, description=description, location=location, when=when, calendar_id=calendar_id)
```
Comment (wiseaidev) [inline code-samples/calendar/python/create_calendar_event/create_calendar_event/__main__.py:128]: There are two ways to create a request body, either using this class, or a dict. The former has an argument `when` of type `nylas.models.events.When` which is not a `dict`, but rather a `Union` type, so not sure how to initialize it.

File: code-samples/calendar/python/create_calendar_event/create_calendar_event/__main__.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        query_params = CreateEventQueryParams(
+            calendar_id="primary", notify_participants=True
+        )
+
+        event, request_id = client.events.create(
```
Comment (wiseaidev) [inline code-samples/calendar/python/create_calendar_event/create_calendar_event/__main__.py:133]: This creates an event, but it doesn't save it to the calendar. Refer to the `main` function to make sure that the event is not created.

Comment (wiseaidev) [reply code-samples/calendar/python/create_calendar_event/create_calendar_event/__main__.py]: Nevermind. This was a bug from my end.

**Comment by wiseaidev** [comment]
This PR is ready for reviews.

**Comment by atejada** [comment]
Congratulations @wiseaidev I will merge this PR. BT, I couldn't find any bugs, the script worked just fine.

**Pull request merged**